src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.ts

```diff
@@ -105,10 +112,17 @@ export class RgwMultisiteWizardComponent implements OnInit {
         .filter((cluster) => cluster['cluster_alias'] !== 'local-cluster');
       this.isMultiClusterConfigured = this.clusterDetailsArray.length > 0;
       if (!this.isMultiClusterConfigured) {
-        this.stepTitles = ['Create Realm & Zonegroup', 'Create Zone', 'Export Multi-site token'];
+        this.stepTitles = ['Create Realm & Zonegroup', 'Create Zone', 'Review']; // If multi-cluster is not configured
```

Comment:
lets have this as a constant/enum and reuse above and here

Reply:
Done. Thanks